### PR TITLE
initial version of audio tagging interface for cacophony-browse

### PR DESCRIPTION
**Pull request by mcuz**

### CODE REVIEW

**Comment by mjs** [comment]
It looks like there's some conflicts with recent work to resolve

**Comment by mcuz** [comment]
Menno I resolved merge conflicts in github directly as opposed to locally.  Seems to be one simple problem that is causing integration tests to fail at this point.  Just to confirm: are you happy for me to do continue doing fixes and rerunning tests then?

File: src/stores/modules/User.store.js [hunk 1]
```diff
@@ -54,8 +54,7 @@ const actions = {
 
     const {result, success} = await api.user.register(payload.username, payload.password, payload.email);
 
-
-    if(success) {
+    if(result.success) {
```
Comment (mjs) [inline src/stores/modules/User.store.js]: Is there a particular reason why was this changed?  The `success` value returned by `register()` indicates that the HTTP status returned for the API request was 2xx. This is what should be used to check if the API call worked. The `success` field on the result object is deprecated and we are going to remove it at some stage.

See also: https://github.com/TheCacophonyProject/cacophony-api/issues/48

**Comment by mjs** [comment]
> Menno I resolved merge conflicts in github directly as opposed to locally. Seems to be one simple problem that is causing integration tests to fail at this point. Just to confirm: are you happy for me to do continue doing fixes and rerunning tests then?

Yep, that's all good. See my review comment about the remaining issue. It seems to me that that particular change should just be reverted.

File: src/stores/modules/User.store.js (hunk 1 above)
Comment (mcuz) [reply src/stores/modules/User.store.js]: yes correct.  I have made that change.

**Comment by mcuz** [comment]
I assume that passing integration tests will automatically deploy?

**Comment by mjs** [comment]
@mcuz not quite, a release is created when a Git tag is created. I do all releases on Monday morning so I'll get it into test then.

**Comment by mjs** [comment]
@mcuz also, You shouldn't have closed the pull request :) I hadn't hit the merge button on it so the code hadn't made it into master. When I hit merge the PR gets closed and marked as such.
